src/app/dropdown/dropdown.component.html

```diff
@@ -59,7 +59,7 @@
           </a>
 
           <a class="menu-item" href="//chat.susi.ai" target="_blank">
-            <div class="img"><img src="../../assets/images/susi.jpg"></div>
+            <div class="img"><img src="../../assets/images/susi_60x12.png"></div>
```

Comment:
Better if we don't change the name of the image and rename the new image to `susi.jpg`.
Also, change PR title from `Fixes#820 Adds new SUSI.AI logo in Dropbox`
to `Fixes#820: Adds new SUSI.AI logo in Dropbox`

Reply:
@ParthS007 Please do the changes as suggested by @aashutoshrathi